_docs/tasks/security/mutual-tls.md

```diff
@@ -43,93 +50,99 @@ Citadel is up if the "AVAILABLE" column is 1.
 
 ### Verifying service configuration
 
-1.  Verify AuthPolicy setting in ConfigMap.
+* Check installation mode. If mTLS is enabled by default (e.g `istio-auth.yaml` was used when installing Istio), it's expected to see uncommented `authPolicy: MUTUAL_TLS` in configmap.
 
     ```command
     $ kubectl get configmap istio -o yaml -n istio-system | grep authPolicy | head -1
     ```
 
-    Istio mutual TLS authentication is enabled if the line `authPolicy: MUTUAL_TLS` is uncommented (doesn't have a `#`).
+* Check authentication policies. mTLS can also be enabled (or disabled) per service(s) by authentication policy. A policy, if exist, will overwrite the configmap setting for the targeted services. Unfortunately, there is no quick way to get relevant policies for a service, other than examining all policies in the applicable namespace:
 
-## Testing the authentication setup
+    ```command
+    $ kubectl get policies.authentication.istio.io -n default -o yaml
+    ```
 
-When running Istio with mutual TLS authentication turned on, you can use curl in one service's
-Envoy to send request to other services.
-For example, after starting the [Bookinfo]({{home}}/docs/guides/bookinfo.html)
-sample application you can ssh into the Envoy container of `productpage` service,
-and send request to other services by curl.
+* Check destination rule. Starting Istio 0.8, destination rule (to be precise, the traffic policy part of destination rule) is used to configure client side to use (or not use) mTLS. For backward compatibility, the _default_ traffic policy is inferred from configmap flag (i.e, if `authPolicy: MUTUAL_TLS`, _default_ traffic policy also be `MUTUAL_TLS`). If there is authentication policy overrules this setting for some services, it should accompany with the appropriate destination rule(s). Similar to authentication policy, the only way to verify the settings is to manually check all rules:
```

Comment:
> destination rule (to be precise, the traffic policy part of destination rule)

API reference is better than words here, IMO. You can just add a link here.
https://istio.io/docs/reference/config/istio.networking.v1alpha3.html#TrafficPolicy